### DISCUSSION TITLE
Bypass auth interceptor for some request

### DISCUSSION THREAD

**Discussion by yeasir-bjit** [cfug/dio#2148] [Q&A]
I have an auth interceptor that adds an access token to every API request. Some of my requests don't need the interceptor. I want to add other headers to those requests. In that case, how do I ignore the interceptor?
The interceptor is added to the dio instance when the app starts.

**Comment by kuhnroyal** [comment]
You can add use an `options.extra['auth'] = true` value and apply the authentication based on that, or the other way around.
